### ISSUE TITLE
NPE on attempting to close Multicast Socket

### ISSUE THREAD

**Issue by markrebhan** [resourcepool/ssdp-client#30]
Version 2.4.3

```
java.lang.NullPointerException: Attempt to invoke virtual method 'void java.net.MulticastSocket.close()' on a null object reference
	at io.resourcepool.ssdp.client.impl.SsdpClientImpl.stopDiscovery(SsdpClientImpl.java:300)
```

**Comment by loicortola** [comment]
Hello!
Thanks, will take a look this weekend

**Comment by loicortola** [comment]
Fixed in 2.4.5

**Comment by VishV-Android** [comment]
Still getting same error.
version 2.5.1

issue created #41
